flyteplugins/go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -35,7 +35,7 @@ const unsignedSIGKILL = 247
 const defaultContainerTemplateName = "default"
 const defaultInitContainerTemplateName = "default-init"
 const primaryContainerTemplateName = "primary"
-const primaryInitContainerTemplateName = "primary-init"
+const PrimaryInitContainerTemplateName = "primary-init"
```

Comment:
These are reserved container names, I don't think they should be exported and used in the spark plugin logic